res/shaders/customLCD.shader/manifest.ini

```diff
@@ -0,0 +1,65 @@
+[shader]
+name=Custom LCD
+author=endrift, Dominus Iniquitatis, EddyHg80
+description=Configurable LCD emulation.
+passes=4
+
+[pass.0]
+fragmentShader=customLCD-brightness.fs
+blend=1
+width=-1
+height=-1
+
+[pass.0.uniform.brightness]
+type=float
+default=1.1
+readableName=Brightness
+
+[pass.1]
+fragmentShader=customLCD-subPixels.fs
+blend=1
+width=-3
+height=-3
```

Comment:
Pass 0 is trivial enough that it should probably also be merged into pass 1.

Reply:
I agree, will do.